test/build_test.go

```diff
@@ -146,3 +148,40 @@ RUN cp /etc/foo /etc/bar
 	wg.Wait()
 	b.StopTimer()
 }
+
+func benchmarkBuildFileTransfer(b *testing.B, sb testutil.Sandbox) {
+	var appliers []fstest.Applier
+	appliers = append(appliers,
```

Comment:
This looks quite stable looking at the results so not a big issue but technically these files do not need to be recreated each time. A global helper func could just create them once and other runs can detect that they already exist. Cleanup can be after `TestMain()` in that case. Looks like it takes around 3 sec atm. I think we can push it a bit more to 5 maybe.

Reply:
Good point will do that